### ISSUE TITLE
Fix redundant links on truss.works

### ISSUE THREAD

**Issue by xtine** [trussworks/accessibility#4]
Issue 8: Redundant links for meet the team, alt text for link incorrect
On the about page, there are two links (Meet the team button and Excited Trussels image) that go to the same destination. For a screen reader, this can be repetitive. Further, the alt text for this image should be describing the function of the link (meet the team page), rather than the image itself.
Issue 9: Redundant Links for blogs
On the blog page, the image and blog title are separate links that go to the same destination. This can be redundant for a user who uses a screen reader
Doc with all documented accessibility issues:
https://docs.google.com/document/d/1upM9ue4UAuJwIBbZ6fRdjGKmzs-odcWxDQfYftIrHyw/edit?usp=sharing

PT IMPORTED STORY: https://www.pivotaltracker.com/story/show/171139000

**Comment by xtine** [comment]
Removed the redundant link to the team page from the "Excited Trussels" image, which should address the alt text issue as well.

**Comment by xtine** [comment]
For blog posts, it seems that the behavior of both the blog image and title being links is a feature that cannot be edited. Not sure how to proceed as it might be set in the theme's code and would not be editable unless we turn on manual HTML editing.

**Comment by xtine** [comment]
Closing this issue because we've done enough as we can without a more complicated solution like code injection for the blog posts. If we can to tackle the blog post links later, we can create a new specific issue for it.
